wordpress-mobile/WordPress-Android
# Issue/2833 single site picker

Addresses issue #2833

Good call, Nick! This change actually made it cleaner.

To test:

1. Go to Site Picker and click search icon. All sites should be shown.
2. Select a site that is hidden. The details page of that site will appear.
3. Going back to the Site Picker, you'll see that the previously hidden site you selected is now visible.
4. Click search icon and select site that is visible. The details page of that site will appear.

Needs review: @nbradbury
